src/app/clusters/resource-monitoring-server/resource-monitoring-server.h

```diff
@@ -161,11 +165,16 @@ class Instance : public CommandHandlerInterface, public AttributeAccessInterface
     ChangeIndicationEnum mChangeIndication         = ChangeIndicationEnum::kOk;
     bool mInPlaceIndicator                         = true;
     DataModel::Nullable<uint32_t> mLastChangedTime;
+    ReplacementProductListManager * mReplacementProductListManager;
```

Comment:
```suggestion
    ReplacementProductListManager * mReplacementProductListManager = nullptr;
```

I think this will fix the segfault from the failed runs